code/mathematical_algorithms/src/smallest_digit_in_number/smallest_digit_in_number.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+
+namespace SmallestNumber
+{
+    public class Program
+    {
+        private static int smallestNumber(int n)
+        {
+            int min = n % 10;
+            n /= 10;
+            
+            while(n!=0)
```

Comment:
Just add spaces around the operators. Also add a space right before opening `(` in all constructs.